exercises/practice/sublist/sublist_test.c

```diff
@@ -0,0 +1,165 @@
+#include "vendor/unity.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+typedef enum {
+    UNEQUAL,
```

Comment:
In the implementation stub you use explicit integers for those. I'm not sure if the C standard guarantees that this starts with 0, but why not be explicit here?
```suggestion
    UNEQUAL = 0,
```

Reply:
You would obviously change that in the generator and not here.